mkt/developers/models.py

```diff
@@ -494,7 +494,7 @@ def to_string(self, type_=None):
 
         try:
             kw = dict(addon=addon, review=review, version=version, group=group,
-                      collection=collection, tag=tag, user=self.user.username)
+                      collection=collection, tag=tag, user=self.user.display_name)
```

Comment:
Everything is returned via self.f on the next line, and the function is actually only used for the log viewer introduced in this patch (its leftover from AMO).